### PR TITLE
feat(rari): support rari as beta feature

### PR DESCRIPTION
**Pull request by fiji-flo**
## Summary

This adds initial support for rari and uses rari to deploy to testing.

### Problem

We want to enable writers and contributors to seamlessly migrate. 

### Solution

Introduce `yarn start:rari` to preview the new system.

This also:
- Fixes the community.md (add a fm with title)
- Stops throwing on unknown flaws

---

## How did you test this change?

Locally and on testing.

### CODE REVIEW

File: server/index.ts [hunk 1]
```diff
@@ -96,19 +114,25 @@ async function buildDocumentFromURL(url: string) {
...
-  return res
-    .status(404)
-    .sendFile(path.join(STATIC_ROOT, "en-us", "_spas", "404.html"));
+async function send404(res: express.Response) {
+  if (!RARI) {
+    return res
+      .status(404)
+      .sendFile(path.join(STATIC_ROOT, "en-us", "_spas", "404.html"));
+  } else {
+    const index = fetch_from_rari("/en-US/404");
```
Comment (argl) [inline server/index.ts:132]:
```suggestion
    const index = await fetch_from_rari("/en-US/404");
```

**Review by argl — APPROVED** [review]
LGTM

File: server/filename.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const filename = import.meta
+  .resolve("./index.ts")
+  .replace(/^file:\/\//, "");
```
Comment (argl) [inline server/filename.ts:3]: Check windows support?

File: tool/popularities.ts [hunk 3]
```diff
@@ -69,7 +69,14 @@ export async function runMakePopularitiesFile({
         pageviews.slice(0, maxUris).forEach(([uri, popularity]) => {
           popularities[uri] = parseFloat(popularity.toFixed(5));
         });
-        fs.writeFileSync(outfile, JSON.stringify(popularities, null, 2));
+        fs.writeFileSync(
```
Comment (argl) [inline tool/popularities.ts:72]: @fiji-flo this implements the updated format

File: package.json [hunk 4]
```diff
@@ -14,36 +15,39 @@
     "yari-tool": "tool/cli.js"
   },
   "scripts": {
-    "ai-help-macros": "cross-env NODE_OPTIONS='--no-warnings=ExperimentalWarning --loader ts-node/esm' node scripts/ai-help-macros.ts",
+    "ai-help-macros": "cross-env NODE_OPTIONS=\"--no-warnings=ExperimentalWarning --loader ts-node/esm\" node scripts/ai-help-macros.ts",
```
Comment (fiji-flo) [inline package.json:18]: These are windows fixes.

**Pull request merged**